packages/steveo/src/common.ts

```diff
@@ -171,7 +172,7 @@ export interface ITask<T = any, R = any> {
   topic: string;
   options: TaskOptions;
   producer: any;
-  publish(payload: T | T[]): Promise<void>;
```

Comment:
@tjcoyoca remember this is an abstraction on top of SQS as well, key isn't used by SQS. What kind of interface for this function signature would be suitable for all backends?

Reply:
my initial thoughts are

```publish(payload: T | T[], context?: { key: string })```
```publish(payload: T | T[], options?: { key: string })```

Reply:
@johndagostino I'd say we go with the first one, `publish(payload: T | T[], context?: { key: string })` as options is already a member of the Task (not that we can't identify this.options vs options but naming wise, I'd like to avoid it as much as possible)